docs/Release_notes.md

```diff
@@ -1,5 +1,139 @@
 CARMA Platform Release Notes
 ----------------------------
+
+Skyline Release, Version 3.1.0, released 21 October 2019 
+--------------------------------------------------------
+**Summary:**
+CARMAPlatform Skyline release version 3.1.0 main highlight is the new Docker configuration and deployment for all the repositories for CARMA3. The images are now available in DockerHub under the organization name of "usdotfhwastol". Docker provides better management of library dependencies, ease of deployment and scalability. Other highlights of this release are the new configurations for the 3 different controllers such as PACMod, NewEagle and DataSpeed, the basic vehicle kinematic model, the updates to the guidance state machine, the updates to the GNSS to map converter, and the new functionality to auto select between GNSS and NDT pose.  
+
+Below are the highlights of the issues and pull requests (PRs) that have been addressed in this release.  
+
+**Repository: CARMAPlatform**  
```

Comment:
Since all issues across the whole system are logged in the CARMAPlatform repo, it might be more appropriate to put this heading just on the list of PRs.  Some of these issues are clearly about content of other repos.

Reply:
Ok will update to move the issues above this header.